go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/rancher/wrangler
 
 go 1.17
 
+replace github.com/rancher/lasso => /Users/rmweir/go/src/github.com/rancher/lasso
```

Comment:
lgtm just need to merge the lasso PR and update this

Reply:
lgtm. will approve once the lasso PR is merged